decisionengine/decisionengine_endpoint/recieve_rock_info.py

```diff
@@ -49,22 +49,23 @@ def find_path():
     userHeightRequest = request.json.get('userHeight')
     if userHeightRequest is None:
         userHeightRequest = '' 
-    elif not isinstance(userHeightRequest, (int, float, complex)):
+    elif not isinstance(userHeightRequest, (int, float)):
         abort(400, 'Bad user height number input')
 
     wallHeightRequest = request.json.get('wallHeight')
     if wallHeightRequest is None:
         wallHeightRequest = '' 
-    elif not isinstance(wallHeightRequest, (int, float, complex)):
+    elif not isinstance(wallHeightRequest, (int, float)):
         abort(400, 'Bad wall height number input')
 
-    endPts = {
-        "rocks":rocks,
-        "userHeight":userHeightRequest,
-        "wallHeight":wallHeightRequest
-    }
-    print(endPts)
-
-    return {'Message': 'Request successfully recieved, congratz on your rockz', 'Response rocks': rocks}
+    endPts = {}
+    add_KV_pair(endPts, 'rocks', rocks)
+    add_KV_pair(endPts, 'userHeight', userHeightRequest)
+    add_KV_pair(endPts, 'wallHeight', wallHeightRequest)
 
+    return endPts
```

Comment:
The message needs to be kept though, can we layout the response like this?

{ response: { endpoints, message }}